cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -249,6 +253,57 @@ func (dp *DevicePlugin) getDpdkMounts(dpdkDeviceName string) []pluginapi.Mount {
 	}
 }
 
+func getDeviceCapabilities(device string) string {
+	devID, err := getDeviceID(device)
+	if err != nil {
+		return defaultCapabilities
```

Comment:
I wonder if there will be any real capabilities if we can't get even the device id...

Reply:
resolved